barretenberg/acir_tests/flows/write_contract.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
I had wrote this flow to easily write contracts against the `acir_artifacts`, however, it is not needed anymore after some CI changes I made. However, it is not unnecessarily useful so I feel we could leave this in?